src/plugins/console/server/lib/spec_definitions/json/generated/indices.disk_usage.json

```diff
@@ -1,24 +1,32 @@
 {
   "indices.disk_usage": {
     "url_params": {
-      "run_expensive_tasks": "__flag__",
-      "flush": "__flag__",
-      "ignore_unavailable": "__flag__",
+      "error_trace": "__flag__",
+      "filter_path": [],
+      "human": "__flag__",
+      "pretty": "__flag__",
       "allow_no_indices": "__flag__",
       "expand_wildcards": [
+        "all",
         "open",
         "closed",
         "hidden",
-        "none",
-        "all"
-      ]
+        "none"
+      ],
+      "flush": "__flag__",
+      "ignore_unavailable": "__flag__",
+      "run_expensive_tasks": "__flag__"
     },
     "methods": [
       "POST"
     ],
     "patterns": [
       "{index}/_disk_usage"
     ],
-    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-disk-usage.html"
+    "documentation": "https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-disk-usage.html",
+    "availability": {
+      "stack": true,
+      "serverless": true
```

Comment:
I'm surprised this one is public too on serverless

Reply:
yes, this one is "internal" in the sheet but not yet updated in the spec